src/components/clusterConfiguration/utils.ts

```diff
@@ -33,36 +53,62 @@ export const getSubnetFromMachineNetworkCidr = (machineNetworkCidr?: string) =>
   if (!machineNetworkCidr) {
     return NO_SUBNET_SET;
   }
-  const subnet = new Netmask(machineNetworkCidr);
-  return `${subnet.toString()} (${subnet.first}-${subnet.last})`;
+
+  const subnet = getSubnet(machineNetworkCidr);
+  return getHumanizedSubnet(subnet);
+};
+
+const getDefaultNetworkSettings = (clusterNetworkCidr: string): ClusterNetworkDefaultSettings => {
+  const isIPv4 = Address4.isValid(clusterNetworkCidr);
+  return {
+    clusterNetworkCidr: isIPv4
+      ? CLUSTER_DEFAULT_NETWORK_SETTINGS_IPV4.clusterNetworkCidr
+      : CLUSTER_DEFAULT_NETWORK_SETTINGS_IPV6.clusterNetworkCidr,
+    serviceNetworkCidr: isIPv4
+      ? CLUSTER_DEFAULT_NETWORK_SETTINGS_IPV4.serviceNetworkCidr
+      : CLUSTER_DEFAULT_NETWORK_SETTINGS_IPV6.serviceNetworkCidr,
+    clusterNetworkHostPrefix: isIPv4
+      ? CLUSTER_DEFAULT_NETWORK_SETTINGS_IPV4.clusterNetworkHostPrefix
+      : CLUSTER_DEFAULT_NETWORK_SETTINGS_IPV6.clusterNetworkHostPrefix,
+  };
 };
 
-export const isAdvConf = (cluster: Cluster): boolean =>
-  cluster.clusterNetworkCidr !== CLUSTER_DEFAULT_NETWORK_SETTINGS.clusterNetworkCidr ||
-  cluster.clusterNetworkHostPrefix !== CLUSTER_DEFAULT_NETWORK_SETTINGS.clusterNetworkHostPrefix ||
-  cluster.serviceNetworkCidr !== CLUSTER_DEFAULT_NETWORK_SETTINGS.serviceNetworkCidr ||
-  cluster.additionalNtpSource !== CLUSTER_DEFAULT_NETWORK_SETTINGS.additionalNtpSource;
+export const getClusterDefaultSettings = (clusterNetworkCidr: string) => ({
+  ...ADDITIONAL_NTP_SOURCE,
+  ...getDefaultNetworkSettings(clusterNetworkCidr),
+});
+
+export const isAdvConf = (cluster: Cluster) => {
+  const defaultNetworkSettings = getDefaultNetworkSettings(cluster.clusterNetworkCidr as string);
```

Comment:
Need to cover `additionalNtpSource` here as well so `getClusterDefaultSettings` needs to be called here